scripts/ceph-windows/setup_ceph_vstart

```diff
@@ -6,7 +6,10 @@ if [[ -z $UBUNTU_SSH_USER ]]; then echo "ERROR: The UBUNTU_SSH_USER env variable
 if [[ -z $UBUNTU_VM_IP ]]; then echo "ERROR: The UBUNTU_VM_IP env variable is not set"; exit 1; fi
 
 export VSTART_DIR="$WORKSPACE/ceph_vstart"
-export VSTART_MEMSTORE_BYTES="5368709120"  # 5GB
+export USE_MEMSTORE=${USE_MEMSTORE:-no}
```

Comment:
Does `USE_MEMSTORE` need to be exported?

Reply:
Whops, thanks.